pkg/virt-controller/watch/migration.go

```diff
@@ -1427,6 +1434,46 @@ func (c *MigrationController) updatePod(old, cur interface{}) {
 	return
 }
 
+// When a resourceQuota is updated, figure out if there are pending migration in the namespace
+// if there are we should push them into the queue to accelerate the target creation process
+func (c *MigrationController) updateResourceQuota(_, cur interface{}) {
+	curResourceQuota := cur.(*k8sv1.ResourceQuota)
```

Comment:
This can be any general update. Do you want to check if the resources were increased?

Reply:
some pods might be deleted as well this could also unblock the migration